modules/article-page/views/index.html

```diff
@@ -15,8 +15,8 @@ <h1>{{ data.page.title }}</h1>
         <h2>
           <a href="{{ article._url }}">{{ article.title }}</a>
         </h2>
-        {% if article.blurb.items and article.blurb.items.length %}
-          {{ article.blurb.items[0].content | safe }}
+        {% if article.blurb.length %}
```

Comment:
Check for `article.blurb` first or we could get a nasty error.

Reply:
No, it's OK, Nunjucks does not error in this situation by design.